concepts/Common/Licensing/10 What is a License Key and How to Get It.md

```diff
@@ -0,0 +1,7 @@
+The license key is meant to help you stay compliant with the EULA of the commercial licenses. While each developer needs to be licensed, the license key is set once per project, where the components are used.
+
+When you purchase a DevExtreme, ASP, DXperience or Universal license, a license key becomes available to you. It can be obtained in [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/).
```

Comment:
Maybe, we could  provide a more detailed instructions for those who never used the DownLoad Manager before